src/app/globals.css

```
@@ -28,6 +28,10 @@
   :root {
     --radius: 0.5rem;
   }
+
+  main a {
```

Should we still restrict this to a className just to keep it opt in?
```suggestion
  .classic-link {
```